superset-frontend/src/views/CRUD/welcome/Welcome.tsx

```diff
@@ -134,8 +134,13 @@ const WelcomeNav = styled.div`
 
 export const LoadingCards = ({ cover }: LoadingProps) => (
   <CardContainer showThumbnails={cover} className="loading-cards">
-    {[...new Array(loadingCardCount)].map(() => (
-      <ListViewCard cover={cover ? false : <></>} description="" loading />
+    {[...new Array(loadingCardCount)].map((_, index) => (
+      <ListViewCard
+        key={index}
```

Comment:
This one seems perfectly fine... it's a static map

Reply:
It will suppress the error warnings though

Reply:
It will suppress the error warnings though

Reply:
Oh yeah, I meant your _fix_ is totally fine - not that it's fine as it was. Thanks for cleaning this up.